asset/quickstart/search-all-iam-policies/main.go

```diff
@@ -51,20 +48,13 @@ func main() {
 		PageSize:  int32(pageSize),
 		PageToken: pageToken,
 	}
-	response := client.SearchAllIamPolicies(ctx, req)
-	for {
-		policy, err := response.Next()
-		if err == iterator.Done {
-			break
-		}
-		if err != nil {
-			log.Fatal(err)
-		}
-		fmt.Println(policy)
-		if response.PageInfo().Remaining() == 0 {
-			break
-		}
+	it := client.SearchAllIamPolicies(ctx, req)
+	var policies []*assetpb.IamPolicySearchResult
+	_, err = iterator.NewPager(it, pageSize, "").NextPage(&policies)
```

Comment:
Any particular reason not to do the for loop? That's what we do for other iterator samples, so it would be nice to be consistent. Is it all guaranteed to fit in a single page?

Reply:
Updated. And removed to unused default fields in the previous comment.
Sorry I've been confusing to you. I wanted to only print the first page instead of all results, but yeah I should be consistent with other samples;)